### ISSUE TITLE
Make Distinction Between "p" and "r" Paths Clearer

### ISSUE THREAD

**Issue by 9p4** [9p4/jellyfin-plugin-sso#84]
**Is your feature request related to a problem? Please describe.**
It seems like there isn't enough of a distinction between the `p` and `r` paths for initialization and redirection respectively. Perhaps make the different clearer?

**Describe the solution you'd like**
Replace the `p` paths with `start` and `r` with `redirect`.

**Describe alternatives you've considered**
Improve the documentation perhaps?

**Additional context**
https://github.com/9p4/jellyfin-plugin-sso/issues/49
https://github.com/9p4/jellyfin-plugin-sso/issues/82
https://github.com/9p4/jellyfin-plugin-sso/pull/55
https://github.com/9p4/jellyfin-plugin-sso/discussions/106

**Comment by 9p4** [comment]
Whatever solution is created, it should NOT break existing configs. `p` and `r` paths should still work, but the documentation should all point to the new paths.

**Comment by esmondmissen** [comment]
How do we want to handle the redirect url changing. Making the routes have multiple paths is easy but the redirect url needs to be configured on the authentication side.

When we get to the code below we have no way of telling if they have configured their redirect url to /r/ or /redirect/.  Could we assume that if they use /start/ we will use /redirect/ and if they use /p/ we will use /r/?

```
var options = new OidcClientOptions
            {
                Authority = config.OidEndpoint?.Trim(),
                ClientId = config.OidClientId?.Trim(),
                ClientSecret = config.OidSecret?.Trim(),
                RedirectUri = GetRequestBase() + "/sso/OID/r/" + provider,
                Scope = string.Join(" ", config.OidScopes.Prepend("openid profile")),
            };
```

**Comment by esmondmissen** [comment]
Something like this:
```
RedirectUri = GetRequestBase() + $"/sso/OID/{(Request.Path.Value.Contains("/start/", StringComparison.InvariantCultureIgnoreCase) ? "redirect" : "r")}/" + provider,
```

**Comment by 9p4** [comment]
I guess there isn't any other option if the auth provider has a whitelist of URLs to redirect to

**Comment by 9p4** [comment]
Fixed in defab9c4e8558d347314bb9a8b74ce673c5dede4
